HW04/anna_tsekhanskaya/lib/student.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'time'
+
+class Student
+  atter_accessor :name, :surname, :noties_for_mentor
+  def initialize
+    @name = name
+    @surname = surname
+    @answer = []
+    @noties_for_mentor = []
+  end
+
+  def full_name
+    "#{name.capitalize} #{surname.capitalize}"
+  end
+
+  # student see notification about new homework
+  # student see notification about homework was reject
+  # student see notification about homework was accept
+  def notifications
+    puts Mentor.noties_for_student
+  end
+
+  # student mark as read all notifications
+  def mark_as_read
+    noties_for_mentor << Notification.new(title: 'Mark as read',
+                                          description: "Student #{Student.full_name} mark as read at #{Time.now}")
+  end
+
+  # student can see homeworks
+  def homeworks
+    Homework.Mentor.see_homework
+  end
+
+  # student take to work homework [1]
+  def to_work(homework)
```

Comment:
If a student takes a homework to work he should add the homework to his homeworks' list